src/utils/tokenizers.py

```diff
@@ -0,0 +1,62 @@
+"""
+Tokenizer class
+To add a tokenizer, add to the below inherting from
+main Tokenizer class.
+
+"""
+import os
+
+from ..openai_transformer_lm.tf_original import utils as openai_utils
```

Comment:
Better to import these in `OpenAIBPETokenizer.__init__()` so they're not loaded if we're not using that model?